changelog.rst

```diff
@@ -23,6 +23,13 @@ Check https://github.com/conan-io/conan for issues and more details about develo
   Please read more :ref:`about Conan stability<stability>`.
 
 
+1.8.0 (##-October-2018)
+-----------------------
+
+- BugFix: ``AutotoolsBuildEnvironment`` packaging library artifacts to ``lib64`` folder in some Linux distros. Setting ``--libdirs`` to
```

Comment:
Bugfix: ``AutotoolsBuildEnvironment`` was not defaulting the output library directory causing broken consumption of packages when rebuilding from sources in different Linux distros using ``lib64`` default.

Reply:
We have to be clear that it was a bug not adjusting the default.